From 823813e882655ad9e92fde4e2f2c71313aa6d261 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 28 Feb 2017 16:09:45 +0100
Subject: [PATCH] bridge/fdb: fix statistics output spacing

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1417289
Upstream Status: iproute2.git commit 85c1807f16774

commit 85c1807f16774286c7fa67090b2fad4d1dae4f18
Author: Scott Feldman <sfeldma@gmail.com>
Date:   Thu Dec 4 09:57:14 2014 +0100

    bridge/fdb: fix statistics output spacing

    Signed-off-by: Scott Feldman <sfeldma@gmail.com>
    Signed-off-by: Jiri Pirko <jiri@resnulli.us>
---
 bridge/fdb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bridge/fdb.c b/bridge/fdb.c
index 97385d0..77f0c77 100644
--- a/bridge/fdb.c
+++ b/bridge/fdb.c
@@ -146,7 +146,7 @@ int print_fdb(const struct sockaddr_nl *who, struct nlmsghdr *n, void *arg)
 		struct nda_cacheinfo *ci = RTA_DATA(tb[NDA_CACHEINFO]);
 		int hz = get_user_hz();
 
-		fprintf(fp, " used %d/%d", ci->ndm_used/hz,
+		fprintf(fp, "used %d/%d ", ci->ndm_used/hz,
 		       ci->ndm_updated/hz);
 	}
 	if (r->ndm_flags & NTF_SELF)
-- 
1.8.3.1

